tests/php/test-class-amp-wordpress-tv-embed-handler.php

```diff
@@ -17,12 +18,17 @@
 class Test_AMP_WordPress_TV_Embed_Handler extends WP_UnitTestCase {
 
 	use AssertContainsCompatibility;
+	use WithoutBlockPreRendering {
+		setUp as public prevent_block_pre_render;
+	}
 
 	/**
 	 * Set up.
 	 */
 	public function setUp() {
 		parent::setUp();
+		$this->prevent_block_pre_render();
```

Comment:
This calls `parent::setUp()` twice, once here, and once in `WithoutBlockPreRendering ::setUp` (which is mapped to `$this->prevent_block_pre_render();` here).